src/bench/CMakeLists.txt

```diff
@@ -46,28 +47,27 @@ add_executable(bench_bitcoin
   strencodings.cpp
   util_time.cpp
   verify_script.cpp
+  $<$<BOOL:${ENABLE_WALLET}>:wallet_balance.cpp>
+  $<$<BOOL:${ENABLE_WALLET}>:wallet_create_tx.cpp>
+  $<$<BOOL:${ENABLE_WALLET}>:wallet_loading.cpp>
   xor.cpp
 )
 
 target_link_libraries(bench_bitcoin
   core
   test_util
-  leveldb
+  bitcoin_node
+  $<TARGET_NAME_IF_EXISTS:bitcoin_wallet>
+  bitcoin_common
```

Comment:
Why the new deps (bitcoin_common, bitcoin_consensus, etc)? Were they necessary before?

Reply:
Otherwise, the "duplicate libraries" warnings still being emitted. Besides, both `test_bitcoin` and `bench_bitcoin` targets do use symbols from the added libraries.

Reply:
I still don't understand. libs are added here but none are removed. How does that eliminate any dupes?

Reply:
[Reworked](https://github.com/hebasto/bitcoin/pull/34#issuecomment-1804024040).